swagger2.0/oic.sec.cred.swagger.json

```diff
@@ -246,8 +245,8 @@
                 "type": "integer"
               },
               "credtype": {
-                 "description": "Representation of this credential's type\nCredential Types - Cred type encoded as a bitmask.0 - Empty credential used for testing1 - Symmetric pair-wise key2 - Symmetric group key4 - Asymmetric signing key8 - Asymmetric signing key with certificate16 - PIN or password32 - Asymmetric encryption key.",
-                 "maximum": 63,
+                 "description": "Represents this credential’s type.\n0 – Used for testing\n1 – Symmetric pair-wise key\n2 – Symmetric group key\n4 – Asymmetric signing key\n8 – Asymmetric signing key with certificate\n16 – PIN or password\n32 – Asymmetric encryption key\n64 - Directly Provisioned OSCORE Security Context",
+                 "maximum": 127,
```

Comment:
Actually, the max value for "credtype" should only be 64 because it is the type of a specific credential, right?

What we need to do is modify "sct" to allow any value between 0 and 127.

Reply:
> Actually, the max value for "credtype" should only be 64 because it is the type of a specific credential, right?
Correct, in "/oic/sec/cred" the maximum should be 64 because it is only the value of a single credtype.
 
> What we need to do is modify "sct" to allow any value between 0 and 127.
Correct, in "/oic/sec/doxm:sct" the maximum should be 127 because any combination of credential types is supported in "/oic/sec/doxm:sct"

Reply:
Great. Who can make this change?  I've never modified files in Github.

Reply:
@marktrayer or @WAvdBeek Github doesn't give me permission to edit OCF files. Could one of you update this "maximum" as follows?

`"maximum": 64,`

Reply:
This is a PR from a branch owned by Oleks.  I can't edit the branch, that's his fork from the OCF master.  We can either ask Oleks to edit and push the commit so it'll be in this PR, or merge the PR then raise a new PR to fix.

Reply:
I support Mark's second option: merge this PR, then raise a new PR to make the corrections.